Sources/NIOEmbedded/Embedded.swift

```diff
@@ -615,10 +629,18 @@ public final class EmbeddedChannel: Channel {
     public var embeddedEventLoop: EmbeddedEventLoop = EmbeddedEventLoop()
 
     /// - see: `Channel.localAddress`
-    public var localAddress: SocketAddress? = nil
+    public var localAddress: SocketAddress? = nil {
```

Comment:
Is there any reason for this not to be a computed property, instead of a `didSet`?